navigation/src/commonMain/kotlin/com/islamversity/navigation/Screens.kt

```diff
@@ -85,4 +85,11 @@ sealed class Screens(
             const val url = "/settings"
         }
     }
+
+    class Licenses : Screens(name,url) {
+        companion object {
+            const val name = "com.islamversity.settings.LicensesView"
+            const val url = "/settings"
```

Comment:
url has to be unique for every "screen",

you can use something like:
`/licenses`